demos/demo_fake_finger.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+"""Basic demo on how to control the Finger robot using a fake driver.
+
+This script illustrates how to control a robot via the Python interface.  It
+uses a fake driver, i.e. it can be executed without an actual robot.  The fake
+driver will simply provide fake observations and ignore any actions that are
+sent to it.
+"""
+import numpy as np
+
+import robot_interfaces
+import blmc_robots
+
+
+def main():
+    # Storage for all observations, actions, etc.
+    finger_data = robot_interfaces.finger.Data()
+
+    # The backend sends actions from the data to the robot and writes
+    # observations from the robot to the data.  Here we use a backend using the
+    # "random finger driver" which just provides fake observations and does not
+    # need an actual robot to be executed.
+    fake_finger_backend = blmc_robots.create_random_finger_backend(finger_data)
+
```

Comment:
on the right hand side you call it random_finger and on the left hand side you call it fake_finger.

my point was just that it should be consistent naming everywhere (including class names, file names, variable names, function names)

Reply:
I see. Given that the `RandomFingerDriver` does not really provide random observations (it is a sequence of increasing values) I would rather rename everything to "fake".  Going to do this in a separate PR, though.